Keas.Mvc/ClientApp/components/Equipment/EquipmentBigFixInfo.tsx

```diff
@@ -127,27 +110,34 @@ export default class EquipmentBigFixInfo extends React.Component<IProps, IState>
             );
         }
 
-        return <p>Not a valid Bigfix id, please make sure to enter a valid Bigfix id.</p>;
+        return <p>No data to present</p>;
     };
 
     private _getBigFixComputerInfo = async (id: string) => {
-        const response = await fetch(`/api/${this.context.team.slug}/equipment/GetComputer/${id}`, {
-            method: "GET"
-        });
-
-        if (!response.ok) {
-            // show the invalid Bigfix-id message.
+        let response = null;
+        try {
+            response = await this.context.fetch(
+                `/api/${this.context.team.slug}/equipment/GetComputer/${id}`
+            );
+        } catch (err) {
+            if (err.message === "Not Found") {
+                toast.error("Not a valid Bigfix id, please make sure to enter a valid Bigfix id.");
```

Comment:
@NabilFurmoli this shouldn't be a toast, the modal should pop up with an error telling the user that their id is invalid.